src/server/dflycmd.cc

```diff
@@ -528,11 +528,7 @@ void DflyCmd::Load(CmdArgList args, RedisReplyBuilder* rb, ConnectionContext* cn
     existing_keys = ServerFamily::LoadExistingKeys::kOverride;
   }
 
-  if (parser.HasNext()) {
-    parser.Error();
```

Comment:
Yeah, don't consume the error if you have extra arguments. We should reject `dfly load path append nonsense` :)